src/Tribe/Views/V2/Theme_Compatibility.php

```diff
@@ -75,6 +77,18 @@ public function filter_add_body_classes( array $classes ) {
 		return array_merge( $classes, $this->get_body_classes() );
 	}
 
+	public function add_body_classes() {
```

Comment:
:no_entry_sign: Missing doc comment for function add_body_classes()

<sub>hash: 41ce5aeee356964850f3e5fb60dc97d7</sub>